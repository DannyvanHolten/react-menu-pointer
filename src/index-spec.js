import * as api from './index';

describe('Index', () => {
  it('should contain the withMenuPointer', () => {
    expect(api).toMatchSnapshot();
  });
});
