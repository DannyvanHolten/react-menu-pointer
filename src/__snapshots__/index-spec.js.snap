// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index should contain the withMenuPointer 1`] = `
Object {
  "default": [Function],
}
`;
